subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -1829,12 +1829,12 @@ void ull_cp_rx(struct ll_conn *conn, struct node_rx_pdu *rx)
 
 #ifdef ZTEST_UNITTEST
 
-static uint16_t local_ctx_buffers_free(void)
+uint16_t local_ctx_buffers_free(void)
```

Comment:
The names of the functions in this file look quite generic to be removing `static`. Did you consider adding a prefix and renaming to something like `ull_local_ctx_buffers_free()`?

Reply:
Good point, even though they are only used for unittest of the module. I'll add the prefix.